docs/contrib-guide/setup-the-project.md

```diff
@@ -37,6 +37,22 @@ cargo install just
 
 :::
 
+- Install `cmake`
+
+You could install just quickly by running the following command or by following the official [Download](https://cmake.org/download/):
```

Comment:
You missed linux, since linux has so much variants, I would suggest just left the official download page link should be enough.